packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesConfiguration.kt

```diff
@@ -132,6 +144,16 @@ class UpdatesConfiguration {
       hasEmbeddedUpdate = hasEmbeddedUpdateFromMap
     }
 
+    val codeSigningCertificateFromMap = map.readValueCheckingType<String>(UPDATES_CONFIGURATION_CODE_SIGNING_CERTIFICATE)
+    if (codeSigningCertificateFromMap != null) {
+      codeSigningCertificate = codeSigningCertificateFromMap
+    }
+
+    val codeSigningMetadataFromMap = map.readValueCheckingType<Map<*, *>>(UPDATES_CONFIGURATION_CODE_SIGNING_METADATA)
```

Comment:
Better to check for `Map<string, string>` explicitly so we don't accidentally get `Map<string, int>` for example?

Reply:
Technically kotlin erases generic types at runtime so these are equivalent, but I'll update it to check for `Map<String, String>` to match the case above.